tasks/updateconstants.js

```diff
@@ -971,27 +970,10 @@ function formatAttrib(attributes, strings, strings_prefix) {
     .filter((a) => a);
 }
 
-function catogerizeItemAbilities(abilities) {
-  const itemAbilities = {};
-  abilities.forEach((ability) => {
-    if (!ability.includes("<h1>")) {
-      (itemAbilities.hint = itemAbilities.hint || []).push(ability);
-    } else {
-      ability = ability.replace(/<[^h1>]*>/gi, "");
-      const regExp = /<h1>\s*(.*)\s*:\s*(.*)\s*<\/h1>\s*([\s\S]*)/gi;
-      try {
-        const [_, type, name, desc] = regExp.exec(ability);
-        (itemAbilities[type.toLowerCase()] =
-          itemAbilities[type.toLowerCase()] || []).push({
-          name: name,
-          desc: desc
-        });
-      } catch (e) {
-        console.log(e);
-      }
-    }
-  });
-  return itemAbilities;
+function categorizeItemAbilities(abilities) {
+  return {
+    hint: cleanupArray(abilities)
```

Comment:
Why is this returning an object instead of just running the function?

Reply:
I kept the original contract of the function, which was returning this object. We can remove the function altogether and construct the object in replaceSpecialAttribs. Let me know what you prefer.

Reply:
that might be easier to read